app/assets/stylesheets/dashboards.scss

```diff
@@ -24,3 +24,28 @@
 .bootstrap_toggle_spacing .checkbox {
   margin-top: 0px;
 }
+
+.pt-phone,.pt-LMP {
```

Comment:
Let's say this -- until we have some design feedback, let's change this to a single class and keep only the padding. I think it does make it look less cramped on the left, which I think would make me feel a little less stressed out if I were CMing!